lib/common/util/constants.js

```diff
@@ -1882,6 +1882,14 @@ var Constants = {
     SNAPSHOT: 'snapshot',
 
     /**
+    * The Share Snapshot value.
```

Comment:
Why it should the comments in upper case? Same to SNAPSHOT, find comments some time in upper case, some time in lower case, would better fix that.

Reply:
Should all be lower case, will fix it.